tooling/unpack-macros.mjs

```diff
@@ -2,47 +2,106 @@
  * Unpack the current `.db` files into human-readable JSON and JS files.
  */
 
-import * as fs from "fs";
-import * as path from "path";
+import fs from "fs";
```

Comment:
Reworked to unpack a v11+ DB into a folder structure. Small additional functional change to remove unnecessary fields from the resulting JSONs.